docs/FAQ.md

```diff
@@ -66,6 +66,58 @@ One session of Zulip Terminal represents a connection of one user to one Zulip s
 
 Since we expect the above to be straightforward for most users and it allows the code to remain dramatically simpler, we are unlikely to support multiple Zulip servers within the same session in at least the short/medium term. However, we are certainly likely to move towards a system to make access of the different servers simpler, which should be made easier through work such as [zulip-terminal#678](https://github.com/zulip/zulip-terminal/issues/678).
 
+## How do I use autocompletes in ZT?
+
+Composing messages could be a tiring task when you have to type all things by yourself. Autocompletes come to rescue here! ZT currently supports some basic autocompletes, for different compose boxes.
+
+### Hotkeys triggering autocomplete
+
+In most places, hitting `tab` allows toggling autocompletes. In ZT, however, `ctrl f` and `ctrl r` are used for toggling purpose:
+* `ctrl f` allows switching between autocomplete suggestions in a forward manner (see the GIF below).
+* `ctrl r` does the same job but in reverse manner.
+
+### Understanding the working
+
+Below are some basic steps to work with autocompletes in ZT (assuming that the user is composing a message):
+1. Type a prefix designated for an autocomplete (e.g., `:` for autocompleting emojis).
+2. Along with the prefix, type the initial letters of the text (e.g., `air` for `airplane`).
+3. Now hit the hotkey. You'd see suggestions being listed in the footer (if there are any).
+4. Cycle between different suggestions as described in above hotkey section. (Notice that a selected suggestion is colored in **black**)
```

Comment:
No, I would say, as we use the default `"code"` palette for this, uniform throughout all themes.

Reply:
It might be simpler to indicate the selected value is highlighted?

We could also add a 5th entry to indicate that cycling "past the end" goes back to the prefix you enter.